### ISSUE TITLE
New Rule: No Subscribe

### ISSUE THREAD

**Issue by timdeschryver** [timdeschryver/eslint-plugin-ngrx#166]
## A snippet of incorrect code

```ts
person: Person;

this.store.select(selectPerson).subscribe(person => {
  this.person = person;
});
```

## A snippet of correct code

```ts
person$ = this.store.select(selectPerson);
```

## I would be willing to submit a PR for the docs :heart:

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No
